pylxd/tests/models/test_container.py

```diff
@@ -431,6 +433,16 @@ def setUp(self):
         super(TestFiles, self).setUp()
         self.container = models.Container.get(self.client, 'an-container')
 
+    @classmethod
+    def setUpClass(TestFiles):
```

Comment:
Putting the temp files in the `setUpClass()` method is probably making the test class do "too much work" as every test in the class will create and tear down the files regardless of whether they are needed.  I suggest that they are moved to the test method, please.